next-env.d.ts

```diff
@@ -1,5 +0,0 @@
-/// <reference types="next" />
```

Comment:
I tihnk you can keep this - it is auto generated but it doesn't add much issue.

Reply:
ok up to you, but didnt you ask me to remove it ?